packages/airnode-node/src/evm/authorization/authorization-fetching.test.ts

```diff
@@ -39,26 +41,6 @@ describe('fetch (authorizations)', () => {
     expect(res).toEqual({});
   });
 
-  it('returns true for all pending requests if authorizers arrays are empty', async () => {
```

Comment:
This is removed because the logic short-circuits the function call right?

Reply:
Correct